nservicebus/upgrades/externalize-encryption.md

```diff
@@ -36,4 +35,23 @@ snippet: SplitDefiningEncryptedPropertiesAs
 
 ## Custom encryption Service
 
-snippet: SplitEncryptionFromIEncryptionService
\ No newline at end of file
+snippet: SplitEncryptionFromIEncryptionService
+
+
+## Compatibility
+
+The NServiceBus.Encryption.MessageProperty package is partially compatible with endpoints using NServiceBus package's encryption functionality:
+
+
+### Sending and receiving using NServiceBus.Encryption.MessageProperty
+* NServiceBus.Encryption.MessageProperty can decrypt and encrypt all messages with message properties of type `NServiceBus.WireEncryptedString`.
+* NServiceBus.Encryption.MessageProperty can decrypt and encrypt all messages with message properties of type `NServiceBus.Encryption.MessageProperty.WireEncryptedString`.
+* NServiceBus.Encryption.MessageProperty can decrypt and encrypt all messages using an encrypted property convention.
+
+
+### Sending and receiving using NServiceBus
+* NServiceBus can decrypt and encrypt all messages with message properties of type `NServiceBus.WireEncryptedString`.
+* NServiceBus can decrypt and encrypt all messages using an encrypted property convention.
+* NServiceBus.Encryption.MessageProperty **cannot** decrypt and encrypt messages with message properties of type `NServiceBus.Encryption.MessageProperty.WireEncryptedString`.
+
+When migrating endpoints to use the NServiceBus.Encryption.MessageProperty package, message contracts should only be updated to use the `NServiceBus.Encryption.MessageProperty.WireEncryptedString` property type when all endpoints are migrated.
```

Comment:
I'd rephrase this section like this:
1. Upgrade all the endpoints to use the new package. The endpoints will continue to process all messages the same way as they've done until then.
2. Upgrade message contracts to use the new package.